R/xctr.R

```diff
@@ -198,6 +198,7 @@ xctr_check <- function(companies, co2) {
   stopifnot(any(grepl("tilt_sector", names(co2))))
   stopifnot(any(grepl("isic_4digit", names(co2))))
   stop_if_col_to_rank_has_missing_values(co2)
+  stop_if_isic_class_not_char("isic_4digit", co2)
```

Comment:
```suggestion
  stop_if_isic_class_not_char(co2, "isic_4digit")
```

Reply:
When creating functios, a common opattern is first data, then details (e.g. column).

https://design.tidyverse.org/args-data-details.html

If `data` is the first argument, your functions can be piped.

```r
data |> 
  stop_if...("some_column") |>
  do_something_if_all_goes_well()
```

Reply:
Totally makes sense! fixed it